prompts/detailed-cautious-epistem-safetyinfo-v1-2507140045-aa7693.txt

```diff
@@ -0,0 +1,108 @@
+`
```

Comment:
dunno if these prompts should be part of the codebase. They seem like they should live somewhere else?